scripts/ebpf_tracing.cmd

```diff
@@ -82,44 +70,93 @@ if "%command%"=="periodic" (
 		mkdir "!traceCommittedPath!"
 	)
 
+	@rem Get the current date and time in a format suitable for appending to file names.
+	for /f "tokens=2 delims==" %%a in ('wmic OS Get localdatetime /value') do (
+		set "dt=%%a"
+		set "YYYY=!dt:~0,4!" & set "MM=!dt:~4,2!" & set "DD=!dt:~6,2!"
+		set "HH=!dt:~8,2!" & set "Min=!dt:~10,2!" & set "Sec=!dt:~12,2!"
+		set "timestamp=!YYYY!!MM!!DD!_!HH!!Min!!Sec!"
+	)
+
     @rem Run down the WFP state.
     pushd "!trace_path!"
     netsh wfp show state
     popd
-    set "wfp_state_file_cab=!trace_path!\wfpstate.cab"
-	makecab "!trace_path!\wfpstate.xml" "!wfp_state_file_cab!"
-	if exist "!wfp_state_file_cab!" (
+    set "wfp_state_file_xml=!trace_path!\wfpstate.xml"
+	if exist "!wfp_state_file_xml!" (
 
 		@rem If the file size is less or equal than 'max_file_size_mb', then move it to the 'traceCommittedPath' directory.
-		for %%F in ("!wfp_state_file_cab!") do (
+		for %%F in ("!wfp_state_file_xml!") do (
```

Comment:
This type of change is not a bug fix, it's an observable design change and so cannot be done in a patch release.  This would require v0.10 being the version number.

Reply:
Theoretically, we are not changing any binaries here, just some scripts, which don't affect the product's functionalities in any way.

Reply:
@dthaler, the script now keeps the default behavior in compressing, while offering the option of not doing so as a parameter.